tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+
+  <!-- border -->![Initial Integration Flow](Basic-Integration-Flow.png)
+
+You must set the following properties:
+
+-	For HTTPS adapter: Under **Connection tab** > **Address**: Define your end point.
+
+  <!-- border -->![The HTTPS settings](HTTPS-Setting.png)
+
+To understand about other settings, refer to [HTTPS Sender Adapter](https://help.sap.com/docs/CLOUD_INTEGRATION/368c481cd6954bdfa5d0435479fd4eaf/0ae4a78909c4479cbc3cc414250919de.html?locale=en-US&version=Cloud)
+
+-	For the JDBC adapter: Under **Connection tab** > **JDBC Data Source Alias**, enter the name of database source to which the adapter should connect to.
+
+  <!-- border -->![The JDBC settings](JDBC-Settings.png)
+
+### Create Sample Database
+
+To execute this scenario, let's create a table with few columns.
+
+1. Add a content modifier and then add following query to its **Message Body**. To ensure duplicate row entry is restricted in your database, make one of the columns as primary key.
+
+    `create table samplejdbc (emp_id integer primary key, emp_name varchar(255));`
+
+      <!-- border -->![Add Content Modifier and input Query](Input-Query.png)
+
+2. Deploy this integration flow. We will now try to execute this via [Postman]( https://www.getpostman.com/) (You can use any other HTTPS client application too.)
+ You can find the endpoint in **Manage Integration Content** section as shows below.
+
+    <!-- border -->![Find Endpoint](Find-Endpoint.png)
+
+    In the **Request** section, enter this **End-point** and **Send** the blank query. This will setup connection of Postman with your integration flow.
+
+    <!-- border -->![Send query from Postman](Blank-Query-Send.png)
+
+3. Before proceeding to the next step, remove Content Modifier from your integration flow. We had added it to create a database successfully.
+
+
+
+> For simplicity, we will use Postman to push the payload, later we will also see how to use Groovy Script to push the payload.
+
+
+### Execute Batch Payload
+
+1. In your integration flow, edit the settings for JDBC Receiver adapter. Under **Connections** tab, select the **Batch Payload** checkbox and **Batch Operation** as **Atomic**. We are doing this because Batch supports XML payload in INSERT, UPDATE and DELETE modes only. Native SQL queries are supported with prepared statements only.
+
+    <!-- border -->![Batch Payload Settings](Batch-Payload-Setting.png)
+
+2. Open your HTTP client (in this case Postman) to create a payload. Enter the **Username** and **Password** under **Authorisation** tab.
+
+    <!-- border -->![Postman- Authorization](Postman-Authorization.png)
+
+3. To create Payload, we will use INSERT mode and create a payload (with table name `samplejdbctest` ) to test Batch Payload scenario. To use these modes, you must follow the syntax. **Send** the following payload from **Request > Body** tab of postman:
+
+    ```
```

Comment:
Please add keyword for code highlighting

Reply:
done